wake_word.py

```diff
@@ -33,26 +40,41 @@ def listen(self):
             print(f"Could not request results from Google Speech Recognition service; {e}")
             return None
 
-    def wait_for_wake_word(self, wake_word="Hey Harry"):
+    def wait_for_wake_word(self, wake_word="Hey Harry", max_attempts=3):
```

Comment:
What is your thinking behind it. Ideally, it should just go back to "sleep" where it can listen to wake word again right? If the code exits, it will be difficult to start it again on a RaspberryPi.

Reply:
yes,if we start the program again and again it is not appropriate as a user experience .will work on that also.